src/Illuminate/Database/Connectors/MySqlConnector.php

```diff
@@ -24,81 +24,55 @@ public function connect(array $config)
         $connection = $this->createConnection($dsn, $config, $options);
 
         if (! empty($config['database'])) {
-            $connection->exec("use `{$config['database']}`;");
+            $connection->exec("USE `{$config['database']}`;");
         }
 
-        $this->configureIsolationLevel($connection, $config);
-
-        $this->configureEncoding($connection, $config);
-
-        // Next, we will check to see if a timezone has been specified in this config
-        // and if it has we will issue a statement to modify the timezone with the
-        // database. Setting this DB timezone is an optional configuration item.
-        $this->configureTimezone($connection, $config);
-
-        $this->setModes($connection, $config);
+        $this->configureConnection($connection, $config);
 
         return $connection;
     }
 
     /**
-     * Set the connection transaction isolation level.
+     * Configure the connection.
```

Comment:
I think this comment could be a lot better, it almost mirrors the method name here

Reply:
@Jamesking56 this reminded me of this particular comment masterpiece: https://github.com/laravel/serializable-closure/blob/master/src/SerializableClosure.php#L51-L56 🤣